### ISSUE TITLE
HA 2024.08 Warning

### ISSUE THREAD

**Issue by mrvanes** [michaelarnauts/home-assistant-comfoconnect#69]
```
Aug 12 21:03:17 kronos hass[659876]: 2024-08-12 21:03:17.109 WARNING (MainThread) [homeassistant.components.fan] Entity None (<class 'custom_components.comfoconnect.fan.ComfoConnectFan'>) does not set FanEntityFeature.TURN_OFF but implements the turn_off method. Please report it to the author of the 'comfoconnect' custom integration
```

**Comment by litinoveweedle** [comment]
fix proposed in PR #70

**Comment by rholligan** [comment]
Getting same logs over here:

> Logger: homeassistant.components.fan
Source: components/fan/__init__.py:295
integration: Fan (documentation, issues)
First occurred: 23 August 2024 at 10:06:13 (2 occurrences)
Last logged: 23 August 2024 at 10:06:13

> Entity None (<class 'custom_components.comfoconnect.fan.ComfoConnectFan'>) does not set FanEntityFeature.TURN_OFF but implements the turn_off method. Please report it to the author of the 'comfoconnect' custom integration
Entity None (<class 'custom_components.comfoconnect.fan.ComfoConnectFan'>) does not set FanEntityFeature.TURN_ON but implements the turn_on method. Please report it to the author of the 'comfoconnect' custom integration

**Comment by michaelarnauts** [comment]
This should be fixed in the 0.2.1 and 0.2.2.
